autopts/wid/pbp.py

```diff
@@ -235,7 +239,6 @@ def hdl_wid_552(params: WIDParams):
     broadcast_code = None
```

Comment:
Can't comment on the lines above, but the broadcast_id should also be part of the TSPX and be fetched the same way, instead of being hard coded here

Reply:
The comment, `# Zephyr stack generates Broadcast_ID itself`, is also no longer true :)

Reply:
I can remove the comment, but this should in the case be done in a whole bunch of the wid modules.

Reply:
Broadcast ID is otherwise only an IXIT value for BAP and TMAP tests, so I think it should be ok to define it in the wid call as done here (similar to the CAP wid file)

Reply:
Done

Reply:
> Broadcast ID is otherwise only an IXIT value for BAP and TMAP tests

PTS being PTS :upside_down_face: